src/transforms/without.ts

```diff
@@ -0,0 +1,41 @@
+import { ComparePredicate } from "../types";
+import { createFilterIterable } from "./filter";
+import { IterableCreatorIterable } from "../IterableCreatorIterable";
+
+export const createWithoutIterable = <TItem>(
+  iterator: Iterable<TItem>,
+  withoutItems: Iterable<TItem>,
+  comparePredicate: ComparePredicate<TItem> | void = void 0
+): Iterable<TItem> => {
+  const withoutSet = new Set(withoutItems);
```

Comment:
This could be moved inside the first `if` branch and just use `withoutItems` in the second branch. This way there's no one extra iteration of the `withoutItems` iterable

Reply:
I believe this is fixed, but for some reason, GitHub is not registering this change.

```ts
export const createWithoutIterable = <TItem>(
  iterator: Iterable<TItem>,
  withoutItems: Iterable<TItem>,
  comparePredicate?: ComparePredicate<TItem>
): Iterable<TItem> => {
  const withoutSet = new Set(withoutItems);
```